docs/assets/less/stacks-documentation.less

```diff
@@ -318,12 +318,15 @@
 //  ----------------------------------------------------------------------------
 
 .stacks-flex-example--item {
-    padding: var(--su8);
-    border: 1px solid var(--bc-medium);
-    background-color: var(--black-075);
-    font-family: var(--ff-mono);
-    font-size: var(--fs-caption);
-    color: var(--fc-dark);
+    &,
+    .s-table td& {
+        padding: var(--su8);
+        border: 1px solid var(--bc-medium);
+        background-color: var(--black-075);
+        font-family: var(--ff-mono);
+        font-size: var(--fs-caption);
+        color: var(--fc-dark);
```

Comment:
The specificity of this selector has increased significantly with this change. The result is a slightly different color for the td text in some of the examples. It is negligible and I think there is no action here.
Before the example text was using `color: var(--fc-medium)` (see line 226 in this file) instead of `--fc-dark`.